docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -110,16 +111,32 @@ public DockerTemplateBase(String image,
 
     protected Object readResolve() {
         if (volumesFrom != null) {
```

Comment:
Your proposed replacement absolutely wrong from data migration view point. Even if current code duplicated, then it not an issue. And as i repeated 100 times personally to you "configuration handling will be changed" no sense in wasting time on code beauty. Instead of continuing unbundling launcher i spent the whole day on unrelated to fixes comments.